snudda/utils/swap_to_degenerated_morphologies.py

```diff
@@ -74,10 +74,17 @@ def write_new_network_file(self):
         print(f"Writing new network to {self.new_network_file}")
         self.new_hdf5 = h5py.File(self.new_network_file, "w")
         self.old_hdf5.copy(source=self.old_hdf5["meta"], dest=self.new_hdf5)
+
+        if len(self.new_snudda_data_dir) > len(self.original_snudda_data_dir):
+            del self.new_hdf5["meta/snuddaData"]
+            self.new_hdf5["meta"].create_dataset("snuddaData", data=self.new_snudda_data_dir)
+        else:
+            self.new_hdf5["meta/SnuddaData"] = self.new_snudda_data_dir
         network_group = self.new_hdf5.create_group("network")
         self.old_hdf5.copy(source=self.old_hdf5["network/neurons"], dest=self.new_hdf5["network"])
 
-        if len(self.new_snudda_data_dir) > len(self.original_snudda_data_dir):
+        # This if statement is now redundant, since we express neuron paths using SNUDDA_DATA
```

Comment:
Perhaps this should be marked for deletion somehow?

Reply:
Removed